examples/gno.land/r/gnoland/pages/page_contribute.gno

```diff
@@ -0,0 +1,93 @@
+
+
+t-shirt size | expected compensation
+-------------|-----------------------
+[XS]         | $ 500
+[S]          | $ 1000
+[M]          | $ 2000
+[L]          | $ 4000
+[XL]         | $ 8000
+_[XXL]_ \*   | $ 16000
+_[3XL]_ \*   | $ 32000
+
+[XS]: https://github.com/gnolang/gno/labels/bounty%2FXS
+[S]: https://github.com/gnolang/gno/labels/bounty%2FS
+[M]: https://github.com/gnolang/gno/labels/bounty%2FM
+[L]: https://github.com/gnolang/gno/labels/bounty%2FL
+[XL]: https://github.com/gnolang/gno/labels/bounty%2FXL
+[XXL]: https://github.com/gnolang/gno/labels/bounty%2FXXL
+[3XL]: https://github.com/gnolang/gno/labels/bounty%2F3XL
+
+\*: XXL and 3XL bounties are exceptional. Almost no issues will have these sizes; most will be broken down into smaller bounties.
+
+## gno.land Grants
+
+The gno.land grants program is to encourage and support the growth of the gno.land contributor community, and build out the usability of the platform and smart contract library. The program provides financial resources to contributors to explore the Gno tech stack, and build dApps, tooling, infrastructure, products, and smart contract libraries in gno.land.
+
+TODO: Add link to new repo
+
+## Join Game of Realms
+
+Game of Realms is the overarching contributor network of gnomes, currently running off-chain, and will eventually transition on-chain. At this stage, a Game of Realm's contribution is comprised of high-impact contributions identified as 'notable contributions'.
+
+These contributions are not linked to immediate financial rewards, but are notable in nature, in the sense they are a challenge, make a significant addition to the project, and require persistence, with minimal feedback loops from the core team.
+
+You can always contribute to the project, and all contributions will be noticed. Contributing now is a way to build your personal contributor profile in gno.land early on in the ecosystem, and signal your commitment to the project, the community, and its future.
+
+There are a variety of ways to make your contributions count:
+- Core code contributions
+- Realm development
+- Validator tooling
+- Developer tooling
+
+How will notable contributions in Game of Realms be selected?
+
+The selection of a notable contribution or the sum of contributions that equal 'notable' is based on the impact it has on the development of the project. For now, it is focused on code contributions, and will evolve over time. The Gno development teams will initially qualify and evaluate notable contributions, and vote off-chain on adding them to the 'notable contributions' folder on GitHub.
+
+To start, we recommend you create an issue in the Game of Realms [repository](https://github.com/gnolang/game-of-realms) to serve as a profile page of all your contributions.`
+
+	_ = b.NewPost("", path, title, body, "2024-06-20T13:17:26Z", nil, nil)
```

Comment:
Do we want to keep the date June 20th 2024?

Reply:
no, we want to update to the latest version's date.

Reply:
Done